lib/include/ert/ecl/ecl_nnc_data.h

```diff
@@ -27,9 +27,14 @@ extern "C" {
 #include <ert/ecl/ecl_nnc_geometry.h>
 #include <ert/ecl/ecl_nnc_export.h>
 
+#define TRANS_DATA     1
```

Comment:
1. Move this to the `ecl_nnc_data` file - no need to let this leak into the public api.
2. Define an `enum` instead of just using `#define`
3. When you use a `2^N` series that is a very clear hint that these values can be used as a bitmask; as long as that is actually not the case I would prefer if you used the values `1,2,3,4`.